olga/analytics/views.py

```diff
@@ -54,13 +104,13 @@ def update_students_without_country_value(active_students_amount, students_per_c
 
         return students_per_country
 
-    def create_instance_data(self, received_data, secret_token):  # pylint: disable=too-many-locals
+    def create_instance_data(self, received_data, access_token):  # pylint: disable=too-many-locals
```

Comment:
This is probably out of the scope of this PR.
But this method must be split into smaller ones and `pylint: disable=too-many-locals` should be removed.